### PR TITLE
[hotfix][docs][QS] MInor cleanup of QS documentation

### PR DESCRIPTION
**Pull request by zentol**
## What is the purpose of the change

This PR fixes a few smaller issues in the Queryable State documentation/javadocs.

### CODE REVIEW

File: docs/dev/stream/state/queryable_state.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ The Queryable State feature consists of three main entities:
  returning it to the client, and 
  3. the `QueryableStateServer` which runs on each `TaskManager` and is responsible for serving the locally stored state.
  
-In a nutshell, the client will connect to one of the proxies and send a request for the state associated with a specific 
+The client will connect to one of the proxies and send a request for the state associated with a specific
```
Comment (greghogan) [inline docs/dev/stream/state/queryable_state.md]: "will connect" -> "connects" (and "send" -> "sends")?

File: docs/dev/stream/state/queryable_state.md [hunk 2]
```diff
@@ -162,14 +161,19 @@ So far, you have set up your cluster to run with queryable state and you have de
 queryable. Now it is time to see how to query this state. 
 
 For this you can use the `QueryableStateClient` helper class. This is available in the `flink-queryable-state-client` 
-jar which you have to explicitly include as a dependency in the `pom.xml` of your project, as shown below:
+jar which you have to explicitly include as a dependency in the `pom.xml` of your project along with `flink-core`, as shown below:
```
Comment (greghogan) [inline docs/dev/stream/state/queryable_state.md]: "you have to explicitly include" -> "must be explicitly included"?

File: docs/dev/stream/state/queryable_state.md [hunk 3]
```diff
@@ -162,14 +161,19 @@ So far, you have set up your cluster to run with queryable state and you have de
...
 <dependency>
   <groupId>org.apache.flink</groupId>
-  <artifactId>flink-queryable-state-client-java_{{ site.scala_version_suffix }}</artifactId>
-  <version>{{site.version }}</version>
+  <artifactId>flink-core</artifactId>
+  <version>{{ site.version }}</version>
+</dependency>
+<dependency>
+  <groupId>org.apache.flink</groupId>
+  <artifactId>flink-queryable-state-client-java{{ site.scala_version_suffix }}</artifactId>
```
Comment (greghogan) [inline docs/dev/stream/state/queryable_state.md]: Do we need to preserve the underscore after `java`?

Comment (zentol) [reply docs/dev/stream/state/queryable_state.md]: no, the underscore is included in `site.scala_version_suffix`.

File: docs/dev/stream/state/queryable_state.md [hunk 4]
```diff
@@ -162,14 +161,19 @@ So far, you have set up your cluster to run with queryable state and you have de
 queryable. Now it is time to see how to query this state. 
 
 For this you can use the `QueryableStateClient` helper class. This is available in the `flink-queryable-state-client` 
-jar which you have to explicitly include as a dependency in the `pom.xml` of your project, as shown below:
+jar which must explicitly included as a dependency in the `pom.xml` of your project along with `flink-core`, as shown below:
```
Comment (greghogan) [inline docs/dev/stream/state/queryable_state.md]: "must" -> "must be"

**Comment by tzulitai** [comment]
+1, LGTM. Will address Greg's last comment and merge this ..

**Pull request closed without merging**